index.html

```diff
@@ -2595,6 +2595,139 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>External Resource Integrity</h2>
+        <p>
+          When including a link to an external resource in a VC, it is 
+          desirable to know whether the resource that is pointed to is the
+          same at signing time as at verification time.  This applies
+          to cases where there is an external resource that is
+          remotely retrieved as well as to cases where the issuer and/or
+          verifier may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
+          context(s) used in the verifiable credential are the same when
+          used by both the issuer and verifier. 
+        </p>
+        <p>
+          To validate that a resource referenced by a Verifiable Credential is
+          the same at verification time as at issuing time, an implementer
+          MAY include a property named <code>resourceIntegrity</code> that
+          stores an array of objects that describe additional integrity
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the [[URL]] to the resource named <code>id</code> and the 
+          <code>integrity</code> information for the resource
+          constructed using the method specified in <a
+          href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource
+          Integrity</a>.
+        </p>
+        <p>
+          There MUST NOT be more than one object in the
+          <code>resourceIntegrity</code> per <code>id</code>. 
+        </p>
+        <p>
+          When performing integrity checking the
+          strongest hash algorithm available SHOULD be used to confirm
+          integrity of the resource.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY
+          contain a property named <code>timestamp</code>
```

Comment:
```suggestion
          contain a property named <code>timestamp</code>
```

-1 for timestamp property, as outlined below. You can probably just presume it's around the signing date for the VC. If we include it, we should be specific about exactly what time of timestamp it is.

Reply:
you ok with language that indicates that the time of hashing should be assumed to be at or before time of vc issuance?

Reply:
Yes, that should work for now... preference for "at the time of VC issuance", possibly with some guidance that implementers should use hashes for resources that have been confirmed to be accurate at the time of VC issuance to help it stick together a bit better.

If we find a common use case where that isn't adequate, we can revisit the `timestamp` property at that time.